### ISSUE TITLE
Intermittent CI error from "window-latest" test

### ISSUE THREAD

**Issue by erights** [endojs/endo#1956]
Several times recently, for the "windows-latest" CI test specifically, I get the error shown at https://github.com/endojs/endo/actions/runs/7481170666/job/20362198293?pr=1860#step:9:2775

```
tool-command › bundle-source command is concurrency safe

  Rejected promise returned by test. Reason:

  Error {
    command: 'node bin/bundle-source --cache-json bundles demo/circular/a.js circular demo/dir1/index.js dir1',
    cwd: 'D:\\a\\endo\\endo\\packages\\bundle-source\\',
    escapedCommand: 'node "bin/bundle-source" --cache-json bundles "demo/circular/a.js" circular "demo/dir1/index.js" dir1',
    exitCode: 1,
    failed: true,
    isCanceled: false,
    killed: false,
    shortMessage: 'Command failed with exit code 1: node bin/bundle-source --cache-json bundles demo/circular/a.js circular demo/dir1/index.js dir1',
    signal: undefined,
    signalDescription: undefined,
    stderr: `(Error#1)␊
    Error#1: EPERM: operation not permitted, rename 'D:\\a\\endo\\endo\\packages\\bundle-source\\bundles\\bundle-dir1-json-meta.json.3064540103.7144.scratch' -> 'D:\\a\\endo\\endo\\packages\\bundle-source\\bundles\\bundle-dir1-json-meta.json'␊
    `,
    stdout: '',
    timedOut: false,
    message: `Command failed with exit code 1: node bin/bundle-source --cache-json bundles demo/circular/a.js circular demo/dir1/index.js dir1␊
    (Error#1)␊
    Error#1: EPERM: operation not permitted, rename 'D:\\a\\endo\\endo\\packages\\bundle-source\\bundles\\bundle-dir1-json-meta.json.3064540103.7144.scratch' -> 'D:\\a\\endo\\endo\\packages\\bundle-source\\bundles\\bundle-dir1-json-meta.json'␊
    `,
  }

  ─

  1 test failed
```

However, when I "rerun failed tests" it goes away, so low priority. However, unlike some other transient CI failures, this one happens often enough to be annoying.

**Comment by kriskowal** [comment]
I have my eye on this too https://github.com/endojs/endo/issues/1955

**Comment by kriskowal** [comment]
Closing as dup of #1955 to be fixed by #1957
